File: 3.0/golang/5.go
package main

import "fmt"

func main() {
	slc := input()
	ans := 0

	for i := 0; i < len(slc)-1; i++ {
		if slc[i] <= slc[i+1] {
			ans += slc[i]
		} else {
			ans += slc[i+1]
		}
	}

	fmt.Println(ans)
}

func input() []int {
	var N int
	fmt.Scanln(&N)

	slc := make([]int, N)
	for i := 0; i < N; i++ {
		var num int
		fmt.Scanln(&num)
		slc[i] = num
	}

	return slc
}

File: 3.0/python/5.py
lst = []
ans = 0

for _ in range(int(input())):
    lst.append(int(input()))

for i in range(len(lst)-1):
    if lst[i] <= lst[i+1]:
        ans += lst[i]
    else:
        ans += lst[i+1]

print(ans)